roles/engineblock/defaults/main.yml

```diff
@@ -64,9 +68,21 @@ engine_minimum_execution_time_on_invalid_received_response: 5000
 engine_time_frame_for_authentication_loop_in_seconds: 60
 engine_maximum_authentication_procedures_allowed: 5
 
-# Regex used to blacklist AuthnContextClassRef attributes from IDP's
-engine_sfo_authn_context_class_ref_blacklist_regex: ''
-
+# This PCRE regex is used to blacklist incoming AuthnContextClassRef attributes on. If an empty string is used
+# the validation is skipped. The validator will throw an exception if the used regex is invalid.
+engine_stepup_authn_context_class_ref_blacklist_regex: '/http:\/\/{{ engine_stepup_base_domain | regex_escape }}\/assurance\/loa[1-3]/'
+# The loa mapping from the internal used LoA's to the Stepup Gateway LOA's
+engine_stepup_engineblock_loa1: "http://{{ base_domain }}/assurance/loa1"
+engine_stepup_engineblock_loa2: "http://{{ base_domain }}/assurance/loa2"
+engine_stepup_engineblock_loa3: "http://{{ base_domain }}/assurance/loa3"
+engine_stepup_gateway_loa2: "http://{{ engine_stepup_base_domain }}assurance/loa2"
+engine_stepup_gateway_loa3: "http://{{ engine_stepup_base_domain }}assurance/loa3"
```

Comment:
These two lines miss a `/` between domain and the word “assurance”

Reply:
Done